01/components/table/TableHeader.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export default function TableHeader() {
+  return (
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Price</th>
+        <th>Quantity</th>
+        <th>Total Price</th>
```

Comment:
Może lepszym rozwiązaniem będzie przekazanie tych informacji przez props?
Może jako tablice, której kolejność ma znaczenie - indeks to numer kolumny